demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -2816,3 +2816,37 @@ def test_run_validation_using_git_on_metadata_with_invalid_tags(
     captured_stdout = std_output.getvalue()
     assert "[PA123]" in captured_stdout, captured_stdout
     assert not res
+
+
+def test_run_validation_using_git_validation_calls(mocker):
```

Comment:
In your test you check only the case `get_changed_files_from_git` returns all sort of files.
Need to add tests for cases it returns also only `old_files`, only `modified_files` ant etc.

Reply:
Added on ee2c841998767b8addaf780f7d76a2ff05ba0d73